src/utils/attendance.json

```diff
@@ -0,0 +1,34 @@
+{
+  "courseName": "Technology",
+  "attendanceCategories": ["P", "PH", "PC", "L", "LE", "NP", "A", "M"],
```

Comment:
I wonder if these values belong here? Do they really change like the rest of the data contained here? If my assumptions is correct (they are not changing), maybe you can defined that array in code and type it.